examples/gno.land/p/demo/grc/grc721/grc721_royalty_test.gno

```diff
@@ -0,0 +1,99 @@
+package grc721
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/testutils"
+	"gno.land/p/demo/users"
+	"gno.land/p/demo/ufmt"
+)
+
+func TestSetTokenRoyalty(t *testing.T) {
+	dummy := NewNFTWithRoyalty(dummyNFTName, dummyNFTSymbol)
+	if dummy == nil {
+		t.Errorf("should not be nil")
+	}
+
+	addr1 := std.Address("g1var589z07ppjsjd24ukm4uguzwdt0tw7g47cgm")
```

Comment:
Same comment as in `metadata_test.gno` about these: consider using the testutils package.

Reply:
I got it https://github.com/gnolang/gno/pull/1962#discussion_r1593361515